install/fo-postinstall.in

```diff
@@ -193,7 +193,11 @@ if [[ $COMMON ]]; then
    chmod 2775 "{$PROJECTSTATEDIR}" "{$PROJECTSTATEDIR}/agents" {$CACHEDIR}
 
    # make sure repo and its parent dir has the right permissions
-   chown {$PROJECTUSER}:$PROJECTGROUP $\{REPO%/*\} $REPO
+   if [[ $(stat -c "%G" $REPO) != {$PROJECTGROUP} ]]; then
+     echo "NOTE: Repository group is not correct."
+     echo "Fixing"
```

Comment:
May be this will gives user better notification. 
```suggestion
     echo "Fixing..This may take long duration, depending on your repository size."
```